<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="chooseBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>260</width>
     <height>70</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>请选择头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="rePasswordLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重新输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="rePasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>670</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>770</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="avatorBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
